sky/skylet/job_lib.py

```diff
@@ -342,8 +365,10 @@ def update_job_status(job_owner: str,
                 # that value. So we take the max here to keep it at RUNNING.
                 status = max(status, original_status)
                 _set_status_no_lock(job_id, status)
-            if need_output:
-                logger.info(f'Updated job {job_id} status to {status}')
+        if not silent:
+            logger.info(f'Updated job {job_id} status to {status}')
```

Comment:
If L339's if is not true, we don't really "update" the original status to status (L338), can we skip this logging?

Reply:
Good catch! Moved the logging back to right place. Thanks!